uint8 object_id
uint8 angle_code
---
bool detected
float32 x_reply
float32 y_reply
float32 z_reply
